docs/api.rst

```diff
@@ -206,6 +206,51 @@ to handle it, which defaults to print a traceback and ignoring the exception.
     errors. To turn a function into a coroutine, they must be ``async def``
     functions.
 
+.. function:: on_automod_rule_create(rule)
```

Comment:
I would move these down towards the guild events rather then having them at the top before client related events.

Reply:
alphabetical

Reply:
This list is not sorted alphabetically..?

Reply:
Why has this been marked as resolved? Nothing has changed